### PR TITLE
💚 feat: add and test build-arg to the build script command (#12)

### LINKED ISSUE #12: Add and test build-arg to the build command

### PR DESCRIPTION
**Pull request by mostafaghadimi**

### CODE REVIEW

**Comment by mostafaghadimi** [comment]
@amir-abdi jan, these changes have been tested on Chinchilla and it works as expected, I have also set the default null value for the build-args not to affect the other services' pipelines.

File: .gitlab-ci-build-openmr.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ build:
     - echo "Building the image"
     - export TAG_NAME=$CI_COMMIT_REF_SLUG-`git show -s --format=%ct HEAD`-`git rev-parse --short=8 HEAD`
     - echo $TAG_NAME
-    - docker build -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$TAG_NAME -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$CI_COMMIT_REF_SLUG-latest .
+    - docker build -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$TAG_NAME -t $ECR_URI_BASE/offerland/$SERVICE_NAME:$CI_COMMIT_REF_SLUG-latest --build-arg SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN:-} --build-arg SENTRY_ENABLED=${SENTRY_ENABLED:-} .
```
Comment (amir-abdi) [inline .gitlab-ci-build-openmr.yml:29]: This env var is not set across all repos; just make sure the docker `--build-args` flag works fine with null/empty values.

Comment (mostafaghadimi) [reply .gitlab-ci-build-openmr.yml:29]: @amir-abdi jan, thanks for your review.
I have checked it locally and on the pipeline and it works fine.

**Review by amir-abdi — APPROVED** [review]
LGTM Mostafa  🚀

**Pull request merged**